src/server/interpreter.py

```diff
@@ -20,19 +20,46 @@
 from cmd import Cmd
 
 from src.parser.parser import Parser
-
+from src.optimizer.statement_to_opr_convertor import StatementToPlanConvertor
+from src.planner.insert_plan import InsertPlan
+from src.parser.table_ref import TableRef, TableInfo
+from src.catalog.models.df_column import DataFrameColumn
+from src.storage.dataframe import load_dataframe
+from src.catalog.column_type import ColumnType
+from src.planner.create_plan import CreatePlan
+from src.executor.create_executor import CreateExecutor
+from src.executor.insert_executor import InsertExecutor
 
 class EvaCommandInterpreter(Cmd):
 
     # Store results from server
     _server_result = None
+    url = None
+
+    def __init__(self):
+        super().__init__()
+
+        # Create table on connecting to server
```

Comment:
Can we add unit tests in `test/server/interpreter`?

Reply:
Moved things to command_handler, so checking there instead.